src/shared-components/Form/Control/Slider.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box } from '@mui/material';
+import Slider from '@mui/material/Slider';
+import { theme } from 'src/styles/theme';
+
+interface CustomSliderProps {
+  defaultValue: number;
+  color: string;
+  thumbbgcolor: string;
+  trackcolor: string;
+  railcolor: string;
+}
+
+export const CustomSlider = ({
+  defaultValue,
+  color,
+  thumbbgcolor,
+  trackcolor,
+  railcolor,
+}: CustomSliderProps) => {
```

Comment:
Remove the other import of the theme, you need to get the theme from context using the `useTheme` hook. Import it form MUI.

```suggestion
}: CustomSliderProps) => {
const theme = useTheme();
```

Reply:
Fixed